src/components/VirtualList/VirtualItem.tsx

```diff
@@ -11,6 +12,7 @@ type VirtualItemProps = {
   index: number;
   config: Resource;
   statefulFilterProps?: React.RefObject<StatefulFilters>;
+  action?: any;
```

Comment:
Actually, after looking more closely, here it seems it could be JSX.Element

Reply:
Yes, and probably I've left a couple of more `any` that I should change in the same way.